spyderlib/config.py

```diff
@@ -512,17 +514,45 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Meta+B",
+              'editor/next char': "Meta+F",
+              'editor/previous word': "Meta+Left",
+              'editor/next word': "Meta+Right",
+              'editor/kill to line end': "Meta+K",
+              'editor/kill to line start': "Meta+U",

```

Comment:
Here is the official list: https://help.ubuntu.com/community/KeyboardShortcuts

In this case, `Super = Windows key`, which is the same as `Meta` for us, right?

I don't see too many conflicts, but it'd be cool to check those that conflict, like `Meta+A`